react/components/organisms/listing/listing.js

```diff
@@ -279,13 +391,42 @@ const styles = StyleSheet.create({
         flex: 1,
         backgroundColor: "#f6f7f9"
     },
-    search: {
+    searchingHeader: {
+        flex: 1
+    },
+    searchingHeaderVertical: {
+        flexDirection: "column",
+        minHeight: 100,
+        marginVertical: 8,
+        marginHorizontal: 0,
+        padding: 0
+    },
+    searchingHeaderHorizontal: {
+        flexDirection: "row",
+        minHeight: 50,
+        marginVertical: 8,
+        marginHorizontal: 15,
+        padding: 0
+    },
+    searchVertical: {
         marginBottom: 10,
         marginHorizontal: 15
     },
-    filters: {
+    searchHorizontal: {
+        width: "50%",

```

Comment:
This `width: 50%` will cause problems if only one is used
![image](https://user-images.githubusercontent.com/25725586/160829778-be402704-d486-48d8-9c39-a90a07a00e82.png)